recipes/newrelic/infrastructure/ohi/sql/ms-sql.yml

```diff
@@ -91,7 +91,19 @@ install:
         - task: assert_infra
         - task: stop_infra
         - task: input_install
+        - task: download
 
+    download:
```

Comment:
Thank you @NSSPKrishna! Please, consider the following comments:

1) The `download` task should run before the `input_install` task for `WebClient` to pick up the right protocols, if needed. So, the order of lines 93 and 94 should be fixed.

2) Rename the `download` task to something like `set_security_protocols` or `set_protocols` to make it more descriptive of what the task is actually doing.

3) Position the `download` task (after being correctly renamed as requested in item 2) to the right place in the code to keep things organized.

Reply:
Made the changes. Please review it again

Reply:
Can you please move all the block of [`set_security_protocols`](https://github.com/newrelic/open-install-library/blob/pk/NR-91116/recipes/newrelic/infrastructure/ohi/sql/ms-sql.yml#L96-L106) to line 407 to keep all tasks code in order inside the file?

Reply:
Refactored to match the order of task.